addons/mail/static/src/models/chat_window/chat_window.js

```diff
@@ -369,6 +387,14 @@ registerModel({
             inverse: 'chatWindow',
             isCausal: true,
         }),
+        componentStyle: attr({
+            compute: '_computeComponentStyle',
+            default: '',
```

Comment:
undefined was making issue?

Reply:
no, haven't though of just using `undefined`, which is actually simpler